lib/events/dynamoevents/dynamoevents.go

```diff
@@ -503,7 +502,12 @@ func (l *Log) SearchEvents(fromUTC, toUTC time.Time, filter string, limit int) (
 
 	var lastEvaluatedKey map[string]*dynamodb.AttributeValue
 	var total int
-	for i := 0; i < backend.DefaultLargeLimit/100; i++ {
+
+	// We iterate over the responses until we roughly hit 100MB in size. This is an arbitrary limit to prevent
+	// runaway loops.
+	// DynamoDB reponses are capped at 900KB according to their documentation.
+	const maxResponseSize = 100 * 1024
```

Comment:
which units is this? If bytes, this needs to be `100*1024*1024` or `100<<20` for 100MiB

Reply:
@pierrebeaucamp could you move it to a const (the same file) and call it something like:

```
pageSizeKbs = 900
maxResponseSizeKbs = 100 * pageSizeKbs
```